src/S3/Result/ListObjectsOutput.php

```diff
@@ -105,6 +105,22 @@ public function getIsTruncated(): ?bool
         return $this->IsTruncated;
     }
 
+    /**
+     * Iterates over Contents and CommonPrefixes.
+     *
+     * @return ObjectAndCommonPrefix[]
```

Comment:
phpstan is not happy with this. Can you help me?

Reply:
How about something like `@return \Traversable<int, ObjectAndCommonPrefix >`?

Reply:
Using `\Traversable<ObjectAndCommonPrefix>` is good enough. Both will work